pkg/dom0-ztools/rootfs/etc/init.d/000-mod-params

```diff
@@ -31,8 +31,6 @@ zfs_set_default_parameters() {
     set_module_parameter zfs zfs_vdev_async_read_max_active 10
     set_module_parameter zfs zfs_vdev_async_write_min_active 1
     set_module_parameter zfs zfs_vdev_async_write_max_active 10
-    set_module_parameter zfs zfs_smoothing_scale 50000
-    set_module_parameter zfs zfs_smoothing_write 5
```

Comment:
Are these now defaults? Or set somewhere else?

Reply:
@eriknordmark 

It's my understanding that they are unneeded with zfs 2.2.0+.

The patch we used '000-Introduce-write-throttle-smoothing.patch'  is discussed here: https://github.com/openzfs/zfs/pull/12868 and some further notes on our options are below:

zfs_smoothing_scale=50000 : "zfs_delay_scale for write smoothing".  controls how quickly the transaction delay approaches infinity.  The default of 100000.
zfs_smoothing_write=5 : which is 5 seconds smoothing should be applied.  The default of 0.

Later in comments of PR 12868 it is mentioned that the below 3 PRs cover it better and the community was in favor of closing the original PR 12868.

https://github.com/openzfs/zfs/pull/13476  which merged to master May 24th, 2022
https://github.com/openzfs/zfs/pull/13838  which merged to master Nov 8, 2022
https://github.com/openzfs/zfs/pull/13839  which merged to master Oct 28, 2022

In particular PR 13476 (included in 2.2.0+) rewrites this path and uses zfs_wrlog_data_max and zfs_dirty_data_max.  It's my understanding that OpenZFS PR 13476 resolves the multi-second long delays which the earlier zfs_smoothing_write 5 second window attempted to mask.

Reply:
Thanks for double-checking